integration-tests/tests/run-catalog-filter.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash/fp')
+const { signIntoTerra, click, clickable, waitForNoSpinners, input, findText, fillIn, heading, findHeading } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+const { dismissNotifications } = require('../utils/integration-utils')
+
+
+const getDatasetCount = async page => {
+  const datasetHeading = await findHeading(page, heading({ level: 2, text: 'Datasets', isDescendant: true }))
+  return datasetHeading.evaluate(node => _.toNumber(_.split('Datasets', node.textContent)[1]))
+}
+
+const testCatalogFilterFn = withUserToken(async ({ testUrl, page, token }) => {
+  const searchText = 'stem cell'
+  const filterItem = 'Granted'
+
+  await page.goto(testUrl)
+  await waitForNoSpinners(page)
+
+  await findText(page, 'Browse Data')
+
+  await page.evaluate(() => window.configOverridesStore.set({ isDataBrowserVisible: true }))
```

Comment:
Will this throw an error when `isDataBrowserVisible` is no longer a valid configuration option? It would be nice if it did so this code would get cleaned up naturally one day.

Reply:
No - but this is a good point. I create [DC-176](https://broadworkbench.atlassian.net/browse/DC-176) to address this issue.

I like the idea of these tests failing once the config setting is no longer needed.